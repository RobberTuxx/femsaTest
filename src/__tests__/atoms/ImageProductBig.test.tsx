import React from 'react';
import renderer from 'react-test-renderer';
import ImageProductBig from "../../components/atoms/imageProductBig";
it('render image product big', ()=> {
    const component = renderer.create(<ImageProductBig url={"https://loremflickr.com/640/480/business"}/>).toJSON()
    expect(component).toMatchSnapshot()
})
