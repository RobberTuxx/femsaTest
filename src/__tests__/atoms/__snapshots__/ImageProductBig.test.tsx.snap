// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render image product big 1`] = `
<View
  style={
    {
      "elevation": 5,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 3.84,
    }
  }
>
  <Image
    source={
      {
        "uri": "https://loremflickr.com/640/480/business",
      }
    }
    style={
      {
        "borderRadius": 10,
        "height": 548,
        "width": 548,
      }
    }
  />
</View>
`;
